README.md

```diff
@@ -12,89 +12,7 @@
-| <a name="input_node_selector"></a> [node\_selector](#input\_node\_selector) | Node selector for Secrets Store CSI Driver pods | `map(any)` | `{}` | no |
-| <a name="input_oidc_provider_arn"></a> [oidc\_provider\_arn](#input\_oidc\_provider\_arn) | OIDC Provider ARN for IRSA | `string` | n/a | yes |
-| <a name="input_release_name"></a> [release\_name](#input\_release\_name) | Helm release name | `string` | `"secrets-store-csi-driver"` | no |
-| <a name="input_resources_driver"></a> [resources\_driver](#input\_resources\_driver) | Driver Resources | `map(any)` | <pre>{<br>  "limits": {<br>    "cpu": "200m",<br>    "memory": "200Mi"<br>  },<br>  "requests": {<br>    "cpu": "200m",<br>    "memory": "200Mi"<br>  }<br>}</pre> | no |
-| <a name="input_resources_liveness"></a> [resources\_liveness](#input\_resources\_liveness) | Liveness Probe Resources | `map(any)` | <pre>{<br>  "limits": {<br>    "cpu": "100m",<br>    "memory": "100Mi"<br>  },<br>  "requests": {<br>    "cpu": "100m",<br>    "memory": "100Mi"<br>  }<br>}</pre> | no |
-| <a name="input_resources_registrar"></a> [resources\_registrar](#input\_resources\_registrar) | Registrar Resources | `map(any)` | <pre>{<br>  "limits": {<br>    "cpu": "100m",<br>    "memory": "100Mi"<br>  },<br>  "requests": {<br>    "cpu": "100m",<br>    "memory": "100Mi"<br>  }<br>}</pre> | no |
-| <a name="input_service_account_name"></a> [service\_account\_name](#input\_service\_account\_name) | Name of service account to create. Not generated | `string` | `"csi-secrets-store-provider-aws"` | no |
-| <a name="input_syncSecretEnabled"></a> [syncSecretEnabled](#input\_syncSecretEnabled) | Sync with kubernetes secrets | `bool` | `false` | no |
-| <a name="input_tolerations"></a> [tolerations](#input\_tolerations) | Tolerations for Secrets Store CSI Driver pods | `list(map(string))` | `[]` | no |
-
-## Outputs
-
-| Name | Description |
-|------|-------------|
-| <a name="output_iam_role_arn"></a> [iam\_role\_arn](#output\_iam\_role\_arn) | ARN of IAM role |
-| <a name="output_iam_role_name"></a> [iam\_role\_name](#output\_iam\_role\_name) | Name of IAM role |
-| <a name="output_iam_role_path"></a> [iam\_role\_path](#output\_iam\_role\_path) | Path of IAM role |
-| <a name="output_iam_role_unique_id"></a> [iam\_role\_unique\_id](#output\_iam\_role\_unique\_id) | Unique ID of IAM role |
- ⎈antares-dev  🌩 default  ~/terraform-aws-secrets-store-csi   main  terraform-docs md .
-## Requirements
-
-| Name | Version |
-|------|---------|
-| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 1.0 |
-| <a name="requirement_aws"></a> [aws](#requirement\_aws) | >= 4.18 |
-| <a name="requirement_helm"></a> [helm](#requirement\_helm) | >= 2.5 |
-
-## Providers
-
-| Name | Version |
-|------|---------|
-| <a name="provider_helm"></a> [helm](#provider\_helm) | >= 2.5 |
+| <a name="provider_helm"></a> [helm](#provider\_helm) | 2.6.0 |
```

Comment:
this should be based `>= 2.5`, based on the `versions.tf`

Reply:
Thanks for spotting; it's fixed now

Reply:
@chrissng This happens because `terraform-docs` took your lock file. You can configure it to ignore lock files.